flagsmith/flagsmith.py

```diff
@@ -272,7 +289,9 @@ def _get_json_response(self, url: str, method: str, body: dict = None):
                 "Unable to get valid response from Flagsmith API."
             ) from e
 
-    def _build_identity_model(self, identifier: str, **traits):
+    def _build_identity_model(
+        self, identifier: str, **traits: typing.Mapping[str, typing.Any]
```

Comment:
Wouldn't hurt to use the flag_engine.identities.traits.types.TraitValue type for the trait values here.